### ISSUE TITLE
Errors on start (Outdated ReflectionHelper)

### ISSUE THREAD

**Issue by Patbox** [InventivetalentDev/PacketListenerAPI#71]
## What steps will reproduce the problem?  
1. Add the plugin
2. Start server

## What were you expecting to happen? What happened instead?  
Just work

## What version of the plugin are you using? *Type /version &lt;Plugin Name&gt;*  
3.7.7-Snap

## What Spigot version are you using? *Type /version*  
git-Paper-144 (MC: 1.16.2)

## What plugins are you using? *Type /plugins*  
![image](https://user-images.githubusercontent.com/39821509/91202514-35653f00-e702-11ea-823a-bbc3bdf60c5b.png)

## Do you have an error log? Use [pastebin.com](http://pastebin.com). *If you're not sure, upload your whole server log* 
```
[18:33:32 INFO]: [PacketListenerApi] Loading PacketListenerApi v3.7.7-SNAPSHOT
[18:33:32 INFO]: [ReflectionHelper/MinecraftVersion] I am loaded from package org.inventivetalent.packetlistener.reflection.minecraft
[18:33:32 WARN]: [ReflectionHelper/MinecraftVersion] Failed to find version enum for 'org.bukkit.craftbukkit.v1_16_R2'/'v1_16_R2'
[18:33:32 INFO]: [ReflectionHelper/MinecraftVersion] Generating dynamic constant...
[18:33:32 INFO]: [ReflectionHelper/MinecraftVersion] Injected dynamic version v1_16_R2 (#11602).
[18:33:32 INFO]: [ReflectionHelper/MinecraftVersion] Please inform inventivetalent about the outdated version, as this is not guaranteed to work.
[18:33:32 INFO]: [ReflectionHelper/MinecraftVersion] Version is v1_16_R2 (11602)
[18:33:32 WARN]: [ReflectionHelper] Failed to find version enum for 'org.bukkit.craftbukkit.v1_16_R2'/'v1_16_R2'
[18:33:32 INFO]: [ReflectionHelper] Generating dynamic constant...
[18:33:32 ERROR]: [org.bukkit.craftbukkit.v1_16_R2.CraftServer] null initializing PacketListenerApi v3.7.7-SNAPSHOT (Is it up to date?)
java.lang.ExceptionInInitializerError: null
	at org.inventivetalent.packetlistener.reflection.resolver.minecraft.NMSClassResolver.resolve(NMSClassResolver.java:15) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ClassResolver.resolveSilent(ClassResolver.java:16) ~[?:?]
	at org.inventivetalent.packetlistener.channel.ChannelAbstract.<clinit>(ChannelAbstract.java:23) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:315) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ClassResolver.resolveObject(ClassResolver.java:35) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ClassResolver.resolveObject(ClassResolver.java:8) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ResolverAbstract.resolve(ResolverAbstract.java:50) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ClassResolver.resolve(ClassResolver.java:27) ~[?:?]
	at org.inventivetalent.packetlistener.ChannelInjector.newChannelInstance(ChannelInjector.java:43) ~[?:?]
	at org.inventivetalent.packetlistener.ChannelInjector.inject(ChannelInjector.java:30) ~[?:?]
	at org.inventivetalent.packetlistener.PacketListenerAPI.load(PacketListenerAPI.java:31) ~[?:?]
	at org.inventivetalent.apihelper.APIManager.registerAPI(APIManager.java:46) ~[?:?]
	at org.inventivetalent.apihelper.APIManager.registerAPI(APIManager.java:65) ~[?:?]
	at org.inventivetalent.packetlistener.PacketListenerPlugin.onLoad(PacketListenerPlugin.java:18) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R2.CraftServer.loadPlugins(CraftServer.java:392) ~[patched_1.16.2.jar:git-Paper-144]
	at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:208) ~[patched_1.16.2.jar:git-Paper-144]
	at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.2.jar:git-Paper-144]
	at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.2.jar:git-Paper-144]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make public java.lang.Object jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(java.lang.Object[]) throws java.lang.InstantiationException,java.lang.IllegalArgumentException,java.lang.reflect.InvocationTargetException accessible: module java.base does not "opens jdk.internal.reflect" to unnamed module @60dcb97c
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340) ~[?:?]
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280) ~[?:?]
	at java.lang.reflect.Method.checkCanSetAccessible(Method.java:198) ~[?:?]
	at java.lang.reflect.Method.setAccessible(Method.java:192) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.util.AccessUtil.setAccessible(AccessUtil.java:56) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.MethodResolver.resolveObject(MethodResolver.java:106) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.MethodResolver.resolveObject(MethodResolver.java:11) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.ResolverAbstract.resolve(ResolverAbstract.java:50) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.MethodResolver.resolve(MethodResolver.java:96) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.resolver.MethodResolver.resolve(MethodResolver.java:90) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.minecraft.Minecraft.newEnumInstance(Minecraft.java:261) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.minecraft.Minecraft$Version.getVersion(Minecraft.java:223) ~[?:?]
	at org.inventivetalent.packetlistener.reflection.minecraft.Minecraft.<clinit>(Minecraft.java:37) ~[?:?]
	... 20 more

```

**Comment by Xmonpl** [comment]
I was just about to send the same error as you, but you overtook me 😂

**Comment by InventivetalentDev** [comment]
Should be fixed in 3.7.8
